test/integration/model/create.test.js

```diff
@@ -942,15 +961,15 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           return User.create({ myvals: [], mystr: [] }, {
             logging(sql) {
               test = true;
-              expect(sql).to.contain('INSERT INTO "UserWithArrays" ("id","myvals","mystr","createdAt","updatedAt") VALUES (DEFAULT,$1,$2,$3,$4)');
+              expect(sql).to.contain('INSERT INTO "UserWithArrays" ("id","myvals","mystr","createdAt","updatedAt") VALUES (DEFAULT,\'{}\',\'{}\'');
```

Comment:
Same thing as above, please assert on the complete SQL, avoid timestamps with `{ timestamps: false }` in the model.